test/TestProjects/ExactMatchFlattenInheritance/Generated/AzureResourceFlattenModel2Container.cs

```diff
@@ -43,26 +43,27 @@ internal AzureResourceFlattenModel2Container(OperationsBase parent) : base(paren
 
         // Container level operations.
 
-        /// <summary> The operation to create or update a AzureResourceFlattenModel2. Please note some properties can be set only during creation. </summary>
```

Comment:
why is the summary removed here?

Reply:
The corresponding operation in the swagger does not have a description
I have added some to this swagger.
But I am not sure if we have more of this kind of cases in the test projects...